### DISCUSSION TITLE
Add Music Assistant tracks / playlists to "Media sources"

### DISCUSSION THREAD

**Discussion by tontonduplaid** [music-assistant/support#1359] [Q&A] [answered]
Hello,
I'm really sorry if this question has already been asked, but I'm just starting out with the Music Assistant beta.

I've added my music provider, YouTube music, and everything works fine in the Music Assistant interface visible in Lovelace.

However, I would like to create an automation to start a lullaby hosted on YT Music every time my daughter goes to bed. But I don't see a setting for finding my Music Assistant playlists in my Home Assistant "Media Sources".

But maybe I don't do things in the right way?
Is there an action for calling up a track / playlist connected to Music Assistant in a Home Assistant automation?

Thank you very much for your help.

**Comment by OzGav** [comment]
You now need to install the MA integration via HACS and then add it via settings>>integrations. Then you can use the mass.queue_command service to call your playlist. Read the FAQs for further details

**Reply by tontonduplaid** [reply]
Hi @OzGav, thank you for your reply.
As I run Home Assistant 2023.07.1, it seems that I can't install Music Assistant as an HACS integration (as it is written in [readme](https://github.com/music-assistant/hass-music-assistant/blob/main/README.md)) 
I can download it, but when I add Music Assistant integration, I have an error message.

So, I had to install [Music Assistant V2 Beta](https://github.com/orgs/music-assistant/projects) as a native Home assistant add-on.

But I don't find a way to use the mass.queue_command service you're talking about.

**Reply by OzGav** [reply]
The addon and HACS integration are different. For your use case you need both. Read all of that readme you linked. Then, as I mentioned, read the FAQs https://github.com/orgs/music-assistant/discussions/710

**Reply by tontonduplaid** [reply]
Thanks for your answer.
Unfortunately, it's impossible for me to add Music Assistant integration.

I guess I can't install it unless I downgrade Home Assistant.
Thanks anyway.

![Capture d’écran 2023-07-10 à 16 08 51](https://github.com/music-assistant/hass-music-assistant/assets/35406480/3e2ee4d6-2bf4-49c3-8f76-04a775372fd4)
![Capture d’écran 2023-07-10 à 16 09 00](https://github.com/music-assistant/hass-music-assistant/assets/35406480/404f3de2-a167-4ac8-af67-2cc451e1cc4c)

**Reply by marcelveldt** [reply, marked as answer]
You need to select in HACS that you want to show beta versions. That way you will be able to install the 2023.6.0b1 version of Music Assistant.

**Reply by tontonduplaid** [reply]
Many thanks @marcelveldt !

I have successfully installed the HACS integration now.

However, unfortunately I am unable to use the service. In my automation, I call the `mass.queue_command` service.

In player_id, I tried the ID of the entity natively integrated by Home Assistant (a speaker with Chromecast integrated: media_player.enceinte_chambre_baby). And I also tried the mass player ID for the same speaker (media_player. enceinte_chambre_baby_2).
In the first case, when running, Home Assistant shows me "Unknown error". In the 2nd case, Home Assistant confirms the execution, but nothing happens.

And I'd prefer to use the native ID so I can control playback from Google Home.

In uri, I put the name of my YouTube Music playlist which is clearly visible in the Home Assistant interface. I tried with and without quotes, but it doesn't work in both cases.

```
service: mass.queue_command
data:
  command: repeat
  player_id: media_player.enceinte_chambre_baby
  uri: "Lullaby for the baby"
  repeat_mode: all
```

**Reply by OzGav** [reply]
You can’t use what you call the native id as this service call has to be passed to the MA player. Use `play_media` as the command

**Comment by OzGav** [comment]
Contains outdated info
